tests/integration/test_syscollector/test_syscollector_configuration.py

```diff
@@ -0,0 +1,434 @@
+t3_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_invalid_configurations.yaml')
+t3_config_parameters, t3_config_metadata, t3_case_ids = get_test_cases_data(t3_cases_path)
+t3_configurations = load_configuration_template(t3_config_path, t3_config_parameters, t3_config_metadata)
+
+# T4 Parameters
+t4_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector_no_tags.yaml')
+t4_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_default_values.yaml')
+t4_config_parameters, t4_config_metadata, t4_case_ids = get_test_cases_data(t4_cases_path)
+t4_configurations = load_configuration_template(t4_config_path, t4_config_parameters, t4_config_metadata)
+
+# T5 Parameters
+t5_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector.yaml')
+t5_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_scanning.yaml')
+t5_config_parameters, t5_config_metadata, t5_case_ids = get_test_cases_data(t5_cases_path)
+t5_config_metadata = t5_config_parameters
+t5_configurations = load_configuration_template(t5_config_path, t5_config_parameters, t5_config_metadata)
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_config_metadata), ids=t1_case_ids)
+def test_syscollector_deactivation(configuration, metadata, set_wazuh_configuration,
+                                   configure_local_internal_options_module, truncate_monitored_files,
+                                   daemons_handler_function):
+    '''
+    description: Check that the module is disabled.
+
+    test_phases:
+        - Configure syscollector.
+        - Configure modulesd in debug mode.
+        - Truncate the log.
+        - Restart analysisd, wazuh-db and modulesd
+
+    wazuh_min_version: 4.4.0
+
+    tier: 0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration loaded from the template located in `configuration` folder.
+        - metadata:
+            type: dict
+            brief: Test case metadata.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set wazuh configuration using the configuration template.
+        - configure_local_internal_options_module:
+            type: fixture
+            brief: Configure the local internal options file.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate the log file before and after the test execution.
+        - daemons_handler_function:
+            type: fixture
+            brief: Handler of Wazuh daemons for each test case.
+
+    assertions:
+        - Check if the syscollector module is disabled.
+
+    input_description:
+        - The `configuration_syscollector.yaml` file provides the module configuration for this test.
+        - The `case_test_syscollector_deactivation.yaml` file provides the test cases.
+    '''
+    file_monitor = FileMonitor(LOG_FILE_PATH) if sys.platform == 'win32' else None
```

Comment:
Try with this pls

```suggestion
    file_monitor = FileMonitor(LOG_FILE_PATH) 
```

Reply:
Done in [515b920](https://github.com/wazuh/wazuh-qa/pull/3584/commits/515b92021581486910bd98c6bdf31d42469bcae5)